src/functions/template-tags/general.php

```diff
@@ -215,7 +215,15 @@ function tribe_get_datetime_format( $with_year = false ) {
 	 * @return mixed|void
 	 */
 	function tribe_get_time_format( ) {
-		$format = get_option( 'time_format' );
+		static $cache_var_name = __FUNCTION__;
```

Comment:
Why is this static?

Reply:
Oh I think I see, but can I get confirmation -- this is to avoid calling __FUNCTION__ over and over which offers *some performance boost*? Was it a worthy boost to look at modifying our use elsewhere?

Reply:
Because the value never changes across calls so assigning only once is a convenient micro-optimization.

Reply:
> Oh I think I see, but can I get confirmation -- this is to avoid calling FUNCTION over and over which offers some performance boost? Was it a worthy boost to look at modifying our use elsewhere?

Bingo! Yeah, it avoids doing the assignment. Probably not worth backporting elsewhere, but worth keeping in mind for future uses.